transports/httpclient/src/test/java/org/commonjava/maven/galley/transport/htcli/internal/HttpDownloadTest.java

```diff
@@ -134,7 +134,7 @@ public void handle( final HttpServletRequest httpServletRequest,
 
         HttpDownload dl =
                 new HttpDownload( url, location, transfer, new HashMap<Transfer, Long>(), new EventMetadata(),
-                                  fixture.getHttp(), new ObjectMapper(), metricRegistry, metricConfig );
+                                  fixture.getHttp(), new ObjectMapper(), metricRegistry, metricConfig, null );
```

Comment:
How about adding another constructor in HttpDownload to avoid to pass null here? I see there are several ones here. :)

Reply:
This is good idea. In fact I wanted to remove the last 3 params which are not needed for test.

Reply:
Yeah, agree.

Reply:
I refected some tests. pls review again.